agents/addon/extension/polly_tts/polly_wrapper.py

```diff
@@ -30,6 +53,21 @@ def __init__(self,
         self.audio_format = 'pcm' # 'json'|'mp3'|'ogg_vorbis'|'pcm'
         self.include_visemes = False
 
+    def validate(self):
+        if not self.voice in VOICE_ENGINE_MAP.keys():
+            raise f"Invalid voice '{self.voice}'. Must be one of {list(VOICE_ENGINE_MAP.keys())}."
```

Comment:
You cannot directly use raise to raise a string. should use an appropriate exception class.
eg:
raise KeyError(f"Invalid voice '{self.voice}'. Must be one of {list(VOICE_ENGINE_MAP.keys())}.")

Also suggest adding error logs

Reply:
Thanks for your review, the raise typo has been corrected in [79e704f](https://github.com/rte-design/ASTRA.ai/pull/200/commits/79e704f5f870909749fe83d397304022dcff97e1), and I've add error logs in [cd1bfad](https://github.com/rte-design/ASTRA.ai/pull/200/commits/cd1bfadfcfe8a93a5be321fae82c4a8f59a9c738) : )